client/my-sites/promote-post-i2/components/tsp-banner/index.tsx

```diff
@@ -17,6 +20,9 @@ function TspBanner( props: TspBannerProps ) {
 		props.onToggle();
 	};
 
+	const site = useSelector( getSelectedSite );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, site?.ID ) );
```

Comment:
This doesn't exclude Atomic sites, so both self-hosted Jetpack sites and WP.com Atomic sites won't open the HC.

You can exclude Atomic sites with the `treatAtomicAsJetpackSite` option:

```suggestion
	const isJetpack = useSelector( ( state ) => isJetpackSite( state, site?.ID, { treatAtomicAsJetpackSite: false } ) );
```